root/entity/Details.js

```diff
@@ -86,17 +86,20 @@ const Details = ({
             <a href={canonicalLink}>{canonicalLink}</a>
           </td>
         </tr>
-        <tr>
-          <th>{l('XML:')}</th>
-          <td>
-            <XMLLink
-              entityGid={entity.gid}
-              entityProperties={entityProperties}
-              entityType={entityType}
-              isSecureConnection={$c.req.secure}
-            />
-          </td>
-        </tr>
+        {/* TODO: show again once genres are implemented at the WS level */}
```

Comment:
Please insert a task ticket reference here.

Reply:
I was planning to do the adding them to the WS today, but I guess I can add a ticket in case this gets released earlier than that PR.

Reply:
Done.

Reply:
Cannot find it. Forgot to force push new commits?

Reply:
Sigh. Yeah. Now really done.

Reply:
Thanks